_includes/setup/android-setup.md

```diff
@@ -38,12 +38,14 @@ you want Flutter to use a different installation of the Android SDK, you must se
 To prepare to run and test your Flutter app on the Android emulator, follow these steps:
 
 1. Enable [VM acceleration](https://developer.android.com/studio/run/emulator-acceleration.html) on your machine.
-1. Launch **Android Studio>Tools>Android>AVD Manager** and select **Create Virtual Device**.
+1. Launch **Android Studio>Tools>Android>AVD Manager** and select
```

Comment:
Note that I do not see this anymore; in Android Studio 3.1 (the current stable), I see the `AVD Manager` entry directly off the `Tools` menu (not in a submenu), and it does show for Flutter projects as well as regular Android ones.

Reply:
Similarly in Android 3.2 Beta.